components/molecules/NivoScatterChart/nivo-scatter-chart.tsx

```diff
@@ -27,21 +35,42 @@ interface ScatterPlotProps {
   }[];
   isMobile?: boolean;
   repositories?: number[];
+  metadata: ScatterChartMetadata;
+  handleSetPrFilter: (state: "open" | "closed" | "all") => void;
 }
+
 const NivoScatterPlot = ({
   data,
   maxFilesModified,
   title,
   setShowBots,
   showBots,
   isMobile,
-  repositories
+  repositories,
+  metadata,
+  handleSetPrFilter
 }: ScatterPlotProps) => {
   const [showMembers, setShowMembers] = useState<boolean>(false);
   const [isLogarithmic, setIsLogarithmic] = useState<boolean>(false);
 
   let functionTimeout: any;
 
+  const handleTogglePrFilter = (val: string) => {
+    switch (val) {
+    case "0":
+      handleSetPrFilter("all");
+      break;
+
+    case "1":
+      handleSetPrFilter("open");
+      break;
+
+    case "2":
+      handleSetPrFilter("closed");
+      break;
+    }
```

Comment:
Can you please add `default` case

Reply:
Intended. The input of the function will always be one of those 3 values. If for any case somehow another value was passed to it, it shouldn't do anything